source/css/scss/vendor_override/foundation/_buttons.scss

```diff
@@ -6,21 +6,19 @@
 button, a.button, .button {
 	font-size: $button-font-sm;
 	font-weight: $font-weight-bold;
-	height: $button-height-med - $button-height-med-reduction;
-	line-height: $button-height-med - $button-height-med-reduction;
-	padding-top: 0;
-	padding-bottom: 0;
+  	padding: rem-calc(12) rem-calc(24);
+  	box-shadow: 0 rem-calc(2) $dark-blue;
 	text-decoration: none;
+  	border-radius: $button-border-radius;
 
 	&:hover, &:active {
 		text-decoration: none;
-		background-color: $kiva-textlink-hover;
+		background-color: $dark-blue;
 	}
```

Comment:
... and if you change the definition of the hover bg color you could use it here